src/wallet/wallet.cpp

```diff
@@ -3495,22 +3495,6 @@ bool CWallet::LoadDestData(const CTxDestination &dest, const std::string &key, c
     return true;
 }
 
-bool CWallet::GetDestData(const CTxDestination &dest, const std::string &key, std::string *value) const
```

Comment:
This is part of the `*DestData` API, please don't remove it just because it's not used at the moment. Same reasoning as here: https://github.com/bitcoin/bitcoin/pull/9987#discussion_r106381766